contracts/addresses/11155111.json

```diff
@@ -1,90 +1,82 @@
+      "gasCompZapper": "0xc1f9de0549d5cd454e1993f04737965893539367",
+      "leverageZapper": "0x294ef77fd242ab52a139bee294ca41bfe1a634df",
+      "collToken": "0xf401c42770cc2489fe9408df0a4bbe94012a4518"
     },
     {
-      "collToken": "0x2e0cf52e8efe8e0059643a71df0303a879997db0",
-      "priceFeed": "0x714a756c198dc340acbf716472831603f97919c0",
-      "addressesRegistry": "0xc6b3fc797b6f70de18361b6e6b83d3453383fd90",
-      "fixedAssetReader": "0xedc8a030311c3785600064c2c4f0c4af32d16b1c",
-      "metadataNFT": "0x049c3259235f02c580ca483ec53bb92ceff689bd",
-      "borrowerOperations": "0xb84b4026818e6cd939c3d919473421d635c78b6b",
-      "troveManager": "0x857c9c207ce7ee248f72d23afe13f2956dde2e60",
-      "troveNFT": "0x10121c1176ec90ed3403ab3fe3ccc4e1172232a5",
-      "stabilityPool": "0xd22e4939c6bdbcfd21b0741986ac9681040ea2e8",
-      "activePool": "0x1a5d27adbb328e42a29c4750610e2fc2c55b5844",
-      "defaultPool": "0x23d356b5132ada27a6c5c8cf1d5b299847b4a3d8",
-      "gasPool": "0x07bff2bd77f5d4dfbef47b8064a76282494f63be",
-      "collSurplusPool": "0xb8e43a845cd709a8ba4caee9315eb6f8b1e1795b",
-      "sortedTroves": "0x9c3886f10c6423091c0ba4ef5f18dcc7f1d964c9",
-      "balancerFlashLoan": "0x1412aea8ec56ec44daa105627c808ff7aa5005f7",
-      "hybridCurveUniV3Exchange": "0x9574a50c9344243dad6cda40d862083c53c198cf",
-      "wethZapper": "0x0f8bff1cf1773fd9ace4e4ca253e74a274d2e37f",
-      "leverageZapper": "0x28368f1d5f77dde5327b63b3234b31339ad95631",
-      "gasCompZapper": "0x535bb531e2a4004ac8108237f9744f4d23101d19"
+      "addressesRegistry": "0xa31e0c20017eef10d5efe8db6b3d2fb0c60157e3",
+      "activePool": "0x7aefb68690f3ea32599dac94c2a300e89115eed9",
+      "borrowerOperations": "0x07dbe5fdf4ac84fa144a7e0bda8c955fd9388d18",
+      "collSurplusPool": "0xceece70a9b5a98de14161d816127265d428fd5c5",
+      "defaultPool": "0xbb80f39a5a7415456401e9368644e4be770cf6b2",
+      "sortedTroves": "0xb9f50e3783a820fde4cb4f3f72d3d8797c9dba14",
+      "stabilityPool": "0x8a92da409531d065ef7c83c239de385eaff8937f",
+      "troveManager": "0xd3868c2b788c5e4fe9714f3e160c7e8c362522a5",
+      "troveNFT": "0x18671024cbcf830ca77b9e90916fbaa022590a0d",
+      "metadataNFT": "0x66a495bf5145d036130de094b037cf50fcd7d39a",
+      "priceFeed": "0xd278ba07bf937e266928973c37fdf1a1fe7be110",
+      "gasPool": "0x4311a25fa634f4165bd39f3ff68c8990826b82bc",
+      "interestRouter": "0xd7eae81178b97e49e6471f398f0486a7c2e43721",
+      "wethZapper": "0x0000000000000000000000000000000000000000",
+      "gasCompZapper": "0xe6c3e2ae319ee41f97c26d8e604beadee7769bdb",
+      "leverageZapper": "0x07c07801507f7c8f47499556828925013a0bbad4",
+      "collToken": "0xd854bf0fb95aee6a18f2f1aecac7df3a705d7a79"
     }
   ],
   "governance": {
-    "LQTYToken": "0x29f6694517cf47208f126da48a32ecbbc8e41e2d",
-    "LUSD": "0x7da678bdcbb2eb6c22d5871735536dcebc4e0dd7",
-    "LQTYStaking": "0xb2062bb31128c4ce026d4a80e90fdf222e8809e5",
```

Comment:
I made LQTYStaking [optional](https://github.com/liquity/bold/pull/655/files#diff-0d00d8e31ce0d61333fa6c8f7afb540d45ef611a95a14df31ed4b6d006bd6e75R45) in the manifest-to-app-env script (I get it from the Governance contract and fill it manually now), but I was wondering removing it was intentional?

Reply:
Definitely not. I have to check what happened here. I saw there’s also 2 constants changed, and I don’t think I touched that.

Reply:
You changed the constants in #562, which was merged last week. It makes sense that the new deployment manifest would get the updated values.

Reply:
As for `LQTYStaking` being removed: my guess is it was only ever included manually. I don't see it ever being present in the code function that generates the governance manifest:
https://github.com/liquity/bold/blob/f8e21aee9bf28b673cd165ccca9a70af9697e764/contracts/scripts/DeployGovernance.s.sol#L235-L249

Reply:
> You changed the constants in https://github.com/liquity/bold/pull/562, which was merged last week.
Ah, right, in my memory this was an old change :sweat_smile: 

> As for LQTYStaking being removed: my guess is it was only ever included manually. 
Good point again! Actually, I have a very rudimentary script that pulls addresses from those auto-generated `run-*.json` files when the script is interrupted for some reason and our final manifest doesn’t get created. There I just pull any created contract, so it becomes more verbose.

Thanks for clarifying it!